doc/man7/openssl-threads.pod

```diff
@@ -55,8 +56,8 @@ L<X509_cmp(3)> takes pointers to C<const> objects, but the implementation
 uses a C cast to remove that so it can lock objects, generate and cache
 a DER encoding, and so on.
 
-Another instance of thread-safety is when an object's internal state,
-such as cached values, are done with locks.
+Another instance of thread-safety is when an updates to an object's
```

Comment:
s/an updates/updates/

Reply:
typo fixed, thanks for the careful reading.